src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClient.cs

```diff
@@ -48,7 +44,7 @@ public ConfigurationClient(string connectionString, HttpPipelineOptions options)
             if (connectionString == null) throw new ArgumentNullException(nameof(connectionString));
             if (options == null) throw new ArgumentNullException(nameof(options));
 
-            _pipeline = options.Build(ComponentName, ComponentVersion);
+            _pipeline = options.Build(typeof(ConfigurationClient));
```

Comment:
What do we do if there are more than one client?

Reply:
Using the same pipeline object? Is this the plan?

Reply:
I can change it to take assembly as an argument.

Reply:
We will definitely have multiple clients sharing one pipeline. It's very common for hierarchical resources like storage account, container, directory, blob/file